src/components/Employee/PaymentMethod/Split.tsx

```diff
@@ -204,24 +198,16 @@ function PercentageField({
   const { t } = useTranslation('Employee.PaymentMethod')
   return (
     <Fragment key={split.uuid}>
-      <NumberField
-        control={control}
+      <NumberInputField
         name={`splitAmount.${split.uuid}`}
-        onChange={e => {
-          onChange(e)
-        }}
-        value={(() => {
-          return percentageValues[split.uuid] ?? 0
-        })()}
         label={t('splitAmountLabel', {
           name: DOMPurify.sanitize(split.name ?? ''),
           account_number: DOMPurify.sanitize(split.hiddenAccountNumber ?? ''),
           interpolation: { escapeValue: false },
         })}
-        formatOptions={{
-          style: 'decimal',
-          currency: currency,
-        }}
+        format="percent"
```

Comment:
Question: this was decimal before, based on the naming tho this seems like the correct update?